templates/S3/synapse-external-bucket.j2

```diff
@@ -166,6 +167,9 @@ Resources:
               Action:
                 - "s3:PutObject"
                 - "s3:PutObjectAcl"
+                # "s3:DeleteObject" returns Conditions do not apply to combination of actions and resources in statement
+                # "s3:*MultipartUpload*" returns  Policy has invalid action
+                # "s3:DeleteObject" returns Conditions do not apply to combination of actions and resources in statement
```

Comment:
@brucehoff This is related to IT-1295.  Would you be able to provide a suggestion on how to allow cross-account grantees the ability to upload objects?